AK/StringView.h

```diff
@@ -28,6 +28,10 @@ class StringView {
         }
     }
     StringView(const AK::String& string);
+    StringView(const AK::StringView& view, int length)
```

Comment:
This strikes me as a weird API for a constructor.
It's more like a "left()" or "substring()" or something, no?

Reply:
Yeah, that's definitely true. I'm not actually sure I ended up needing it anyway.

Reply:
Okay, I did, but I've replaced it by using the char*, len ctor for now. I think that some left/right/mid sugar would be nice, but I'd rather do that as a separate change on top.